pkg/service/connections.go

```diff
@@ -71,10 +77,54 @@ func (s *Service) TerminateUDPConnectionsToBackend(l3n4Addr *lb.L3n4Addr) {
 			log.Errorf("Forcefully terminating sockets connected to deleted service backends " +
 				"not supported by underlying kernel: see kube-proxy free guide for " +
 				"the required kernel configurations")
+			return
 		} else {
-			log.Errorf("Error while forcefully terminating sockets connected to"+
-				"deleted service backend: %v. If you see any traffic going to such"+
-				"deleted backends, consider restarting application pods sending the traffic.", err)
+			log.WithError(err).WithField(logfields.L3n4Addr, l3n4Addr).Error(
+				"error while forcefully terminating sockets connected to" +
+					"deleted service backend. Consider restarting any application pods sending traffic" +
+					"to the backend")
+		}
+	}
+
+	// Iterate over all pod network namespaces, and terminate any stale connections.
+	if option.Config.EnableSocketLBPodConnectionTermination && !option.Config.BPFSocketLBHostnsOnly {
```

Comment:
Why does this depend on `BPFSocketLBHostnsOnly`?

Reply:
>Why does this depend on BPFSocketLBHostnsOnly?

`BPFSocketLBHostnsOnly`: Socket-LB is enabled only for the host netns.

Reply:
And what about hostNS pods that have stale UDP connections? Wouldn't we still want to terminate those?